lampshade.go

```diff
@@ -187,41 +222,45 @@ const (
 	winSize        = 4
 	maxPaddingSize = 1
 	tsSize         = 8
+	maxSecretSize  = 32
```

Comment:
Oh, I see. The init message has 32 bytes for secret, but how it's used depends on the cipher.